### PR TITLE
Dhub 34: Toevoegen rdf uri

### PR DESCRIPTION
**Pull request by ahup**

### CODE REVIEW

**Review by illuminated-roel — APPROVED** [review]

File: core/src/main/java/org/dotwebstack/framework/core/datafetchers/FieldConstants.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.dotwebstack.framework.core.datafetchers;
+
+public class FieldConstants {
+  private FieldConstants() {}
+
+  public static final String RDF_URI_FIELD = "_id";
+
```
Comment (kad-simonm) [inline core/src/main/java/org/dotwebstack/framework/core/datafetchers/FieldConstants.java]: Remove blank line

File: core/src/test/java/org/dotwebstack/framework/core/datafetchers/GenericDataFetcherTest.java [hunk 2]
```diff
@@ -190,9 +297,110 @@ private DataFetchingEnvironment createDataFetchingEnvironment(GraphQLOutputType
...
+  private Map<String, Object> createDataMap(String key1, Object value1, String key2, Object value2) {
+    Map<String, Object> dataMap = new HashMap<>() {
+      {
+        put(key1, value1);
+        put(key2, value2);
+      }
+    };
+    return dataMap;
+  }
+
```
Comment (kad-simonm) [inline core/src/test/java/org/dotwebstack/framework/core/datafetchers/GenericDataFetcherTest.java:404]: Remove blank line

File: integration-test/graphql-postgres/src/test/java/org/dotwebstack/framework/integrationtest/graphqlpostgres/GraphQlPostgresIntegrationTest.java [hunk 3]
```diff
@@ -641,4 +647,75 @@ void graphQlQuery_returnsBreweryWithPostalAddressAndUnknownVisitAddress_forBrewe
...
+    assertTrue(data.containsKey("brewery"));
+    Map<String, Object> brewery = ((Map<String, Object>) data.get("brewery"));
+    assertFalse(brewery.containsKey("_id"));
+    assertTrue(brewery.containsKey("visitAddress"));
+    assertThat(brewery.get("visitAddress"), nullValue());
+
+    assertPostalAddressHasRdfUri(brewery);
+
+    assertBeersHaveRdfUri((List<Map<String, Object>>) brewery.get("beers"));
+
```
Comment (kad-simonm) [inline integration-test/graphql-postgres/src/test/java/org/dotwebstack/framework/integrationtest/graphqlpostgres/GraphQlPostgresIntegrationTest.java]: Remove blank line

File: core/src/test/java/org/dotwebstack/framework/core/datafetchers/SelectionSetWrapperTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    List<SelectedField> fields = selectionSetWrapper.getFields("*.*");
+    assertThat(fields.size(), is(1));
+    assertFalse(containsFieldWithName(fields, "_id"));
+  }
+
+  private boolean containsFieldWithName(List<SelectedField> fields, String name) {
+    return fields.stream()
+        .anyMatch(field -> field.getName()
+            .equals(name));
+
```
Comment (kad-simonm) [inline core/src/test/java/org/dotwebstack/framework/core/datafetchers/SelectionSetWrapperTest.java]: Remove blankline

**Review by kad-simonm — APPROVED** [review]

**Pull request merged**